vault/diagnose/file_checks_windows.go

```diff
@@ -0,0 +1,10 @@
+// +build windows
+
+package diagnose
+
+import "io/fs"
+
+// IsOwnedByRoot does nothing on windows
+func IsOwnedByRoot(info fs.FileInfo) bool {
```

Comment:
Lets add a TODO here that there may be some similar check on Windows.  But we can worry about it later.

Reply:
Sounds good!